src/main/scala/exercises/06_Collections.scala

```diff
@@ -78,9 +82,22 @@ object Collections {
    *  - custom made
    *  - ... etc 
    */
-  def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
-  }
+  def addElement[T](l: List[T], e:T) = l:+ e
+ 
+  
+  def concatLists[T](l1: List[T], l2: List[T]): List[T] = 
+    l1 ::: l2;
+ 
+
+    // as you can tell, one thing I wanted to do was to use a for loop
+    // to add the element at the nth index of l2 to l1
+    // would appreciate help about how to make this work
+    // val a = 0;
+    // for (a <- 1 to l2.size) { 
+    // addElement(l1, l2(a))
+    // }
```

Comment:
Didn't actually test this with code, but I think two improvements are getting rid of `val a = 0;` because for-loops in Scala don't need initialization beforehand. It's similar to Python with its `for i in range(___)`. In addition,  for solving this problem imperatively, you need to update `l1` as it keeps adding more elements. Currently, `addElement` doesn't update `l1`.